src/ComponentBase.ts

```diff
@@ -391,6 +399,40 @@ export abstract class ComponentBase<P extends React.Props<any>, S extends Object
         });
     }
 
+    /**
+     * search Subscription "keyPropertyName" in Component props(this.props)
+     *
+     * @private
+     * @param {Readonly<P>} props
+     * @param {string} keyPropertyName
+     *
+     * @return {string}
+     */
+    private _findKeyPropertyName(props: Readonly<P>, keyPropertyName: string): string {
+        const key: string | undefined = _.get(this.props, keyPropertyName);
```

Comment:
I think you meant to use props instead of this.props

Reply:
Yes, it is a typo. I fixed it.